testsuite/features/support/env.rb

```diff
@@ -275,6 +303,7 @@ def process_code_coverage
 
 # Create a user for each feature
 Before do |scenario|
+  check_read_timeout_threshold
```

Comment:
I think we don't need to run it here, as we are already running in another `Before` method. If we add it here too, we will repeating that check multiple times before running a scenario.